app/test/unit/git/apply-test.ts

```diff
@@ -115,8 +117,8 @@ describe('git/apply', () => {
       const diff = await getDiff(filePath)
 
       // Check that the file has no local changes.
-      expect(diff.text).toEqual('')
-      expect(diff.hunks).toEqual([])
+      assert.equal(diff.text, '')
+      assert(diff.hunks.length === 0)
```

Comment:
Nit: Seeing going back and forth on `assert(length == x)` and `assert.equal(length, x)` ... very nittish.. but consistently doing one would be nice. (I favor `assert.equal(length, x)`)

Reply:
Addressed in dcec329f1c7229a74377f189af41c99e5aebdc69